hardware/cpu/intel/xeon_E5520_vm_singlecore.pan

```diff
@@ -0,0 +1,8 @@
+structure template hardware/cpu/intel/xeon_E5520_vm_singlecore;
```

Comment:
These seem like they are very site specific and probably don't belong in the standard library.